docs/src/main/tut/quickstart.md

```diff
@@ -25,7 +25,7 @@ It's divided into multiple and different artifacts, grouped by scope:
 --- | --- | --- | ---
 `frees-rpc-server` | Server | Yes | Needed to attach RPC Services and spin-up an RPC Server.
 `frees-rpc-client-core` | Client | Yes | Mandatory to define protocols and auto-derived clients.
-`frees-rpc-client-netty` | Client | Yes* | Optional if you use `OkHttp`, required from the client perspective.
+`frees-rpc-client-netty` | Client | Yes* | Mandatory on the client side if we are using Netty in the server side.
```

Comment:
Thanks for updating - could you confirm if it's mandatory for the client implementation to match the server's though?

Also we'll want to adjust the following line in a similar way, for `OkHttp`. Thanks!

Reply:
Yes, the matching it's mandatory.